src/shared/components/case-viewer/case-review-specific-access-request/case-review-specific-access-request.component.scss

```diff
@@ -0,0 +1,3 @@
+.left-label {
+  width: 220px
+}
```

Comment:
Complaining about no newline. Also, is there any particular reason why the left-label is a magic number?